core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditor.js

```diff
@@ -168,8 +170,17 @@ oppia.factory('trainingModalService', [
                   editorContextService.getActiveStateName();
                 var state = explorationStatesService.getState(currentStateName);
 
+                // Retrieve the interaction ID.
+                var interactionId = stateInteractionIdService.savedMemento;
+
+                var rulesServiceName = interactionId.charAt(0).toLowerCase() +
+                interactionId.slice(1) + 'RulesService';
```

Comment:
Indent this line by 2.

Do you think it's worth breaking out a getNameOfRulesService(interactionId) method?